css/styles.css

```diff
@@ -0,0 +1,116 @@
+    
+    header{
+
+        text-align:left;
+        height:20%;
+        padding-left:30px;
+        display:inline-block;
+
+    }
+
+    photos{
+
+        height:150px;
+        width:830px;
+        display:inline-block;
+        padding-top: 10px;
+        padding-bottom:10px;
+        padding-left:30px;
+        padding-right:10px;
+        margin-bottom:30px;
+        margin-left:30px;
+        clear:left;
```

Comment:
Try declaring `clear: both;` instead. It is essentially the same as floating left and right.